### PR TITLE
Fix forgotten code smells from issue 115

### PR DESCRIPTION
**Pull request by islean**
## Description

When merging and deploying the fix for issue 115, I missed fixing some code smells which this pr should take care off.

### Changed

- Fixtures which were named "context" but were actually dictionaries are now type hinted as such.
- Constant added for string "new file added"
- Removed some variable assignments and cleaned up some return statements.

### How to prepare for test

- [x] ssh to hasta (depending on type of change)
- [x] install on stage:
```bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t housekeeper -b fix-forgotten-code-smells-126```

### How to test
- [x] Run ```housekeeper --help```

### Expected test outcome
- [x] check that housekeeper is not down, i.e. that the CLI is up and running.
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: tests/cli/test_cli_include.py [hunk 1]
```diff
@@ -76,22 +72,18 @@ def test_include_files_adds_version_specific_files(
     # GIVEN that the latest version of the bundle is not included
     assert version_obj.included_at is None
     # GIVEN that no version specific folder has been created since version is not included
-    version_path = populated_context[ROOT] / bundle.name / str(version_obj.created_at.date())
+    version_path = Path(populated_context[ROOT], bundle.name, str(version_obj.created_at.date()))
```
Comment (henrikstranneheim) [inline tests/cli/test_cli_include.py:75]: 👍

**Review by henrikstranneheim — APPROVED** [review]
Well done 👍

File: housekeeper/constants.py [hunk 2]
```diff
@@ -8,3 +8,4 @@
 ARCHIVE_TYPES = ("data", "result", "meta", "archive")
 EXTRA_STATUSES = ["coverage", "frequency", "genotype", "visualizer", "rawdata", "qc"]
 ROOT: str = "root"
+NEW_FILE_ADDED: str = "new file added"
```
[the remark this answers is not in the document]
Comment (islean) [reply housekeeper/constants.py]: Nice spotting!

**Review by seallard — APPROVED** [review]

**Comment by islean** [comment]
Test okay:

image

**Pull request merged**